lib/vaahextendflutter/widgets/atoms/input_text.dart

```diff
@@ -48,19 +49,20 @@ class InputText extends StatelessWidget {
     this.suffixOnTap,
     this.minLines,
     this.maxLines,
+    this.inputBorder = InputBorderType.outline,
   });
 
   @override
   Widget build(BuildContext context) {
     return TextFormField(
       decoration: InputDecoration(
         contentPadding: padding,
-        border: border(AppTheme.colors['black']!.shade400),
-        enabledBorder: border(AppTheme.colors['black']!.shade400),
-        disabledBorder: border(AppTheme.colors['black']!.shade300),
-        focusedBorder: border(AppTheme.colors['black']!.shade400),
-        errorBorder: border(AppTheme.colors['danger']!.shade400),
-        focusedErrorBorder: border(AppTheme.colors['danger']!.shade400),
+        border: getInputBorder(inputBorder, AppTheme.colors['black']!.shade400),
+        enabledBorder: getInputBorder(inputBorder, AppTheme.colors['black']!.shade400),
+        disabledBorder: getInputBorder(inputBorder, AppTheme.colors['black']!.shade400),
```

Comment:
why did we change shade?

Reply:
It happened by mistake due to copy and paste same function.